backend/routes/stripe.js

```diff
@@ -194,18 +199,20 @@ module.exports = function (router) {
 
   router.post('/stripe/check-creds', authShop, async (req, res) => {
     let valid = false
+    const { stripeKey, stripeBackend } = req.body
 
-    try {
-      const { stripeBackend } = req.body
-
-      const stripe = Stripe(stripeBackend)
+    if (validatePublishableKey(stripeKey) && validateSecretKey(stripeBackend)) {
```

Comment:
Nit: Perhaps we could add a check to make sure that both are either live or test keys and not mixed. But, don't think it's a problem

Reply:
Good idea.  Gonna do a little refactor to include this check.  Thanks